### PR TITLE
Vetext twilio incoming forwarder lambda

### PR DESCRIPTION
**Pull request by jakehova**
VeText Twilio Incoming Messages Lambda Handler

### CODE REVIEW

File: .github/workflows/lambda-functions.yaml [hunk 1]
```diff
@@ -36,7 +37,7 @@ jobs:
       - name: Set Python version
         uses: actions/setup-python@v2
         with:
-          python-version: '3.6'
+          python-version: "3.6"
```
Comment (kalbfled) [inline .github/workflows/lambda-functions.yaml:40]: Should this be 3.8?

File: lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+def vetext_incoming_forwarder_lambda_handler(event: any, context: any):
+    """this method takes in an event passed in by either an alb or sqs.
+        @param: event   -  contains data pertaining to an incoming sms from Twilio
+        @param: context -  contains information regarding information
+            regarding what triggered the lambda (context.invoked_function_arn).
+    """
+
+    try:
+        print(event)
```
Comment (kalbfled) [inline lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py:22]: General comment:  Change all the print statements to log statements.

**Review by jessecanderson — APPROVED** [review]

File: lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+        # Determine if the invoker of the lambda is SQS or ALB
+        #   SQS will submit batches of records so there is potential for multiple events to be processed
+        #   ALB will submit a single request but to simplify code, it will also return an array of event bodies
+        if (event["requestContext"] and event["requestContext"]["elb"]):
```
Comment (kalbfled) [inline lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py:27]:
```
if "requestContext" in event and "elb" in event["requestContext"]:
```

Alternatively . . .
```
if "elb" in event.get("requestContext", {}):
```

File: lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+            print("alb invocation")
+            event_bodies = get_body_from_alb_invocation(event)            
+        elif event["Records"] and event["Records"][0].eventSource == 'aws:sqs':
```
Comment (kalbfled) [inline lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py:30]:
```
elif "Records" in event and event["Records"][0].eventSource == 'aws:sqs':

```

File: lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+            print("sqs invoication")
+            event_bodies = get_body_from_sqs_invocation(event)
+        else:
+            return{
```
Comment (kalbfled) [inline lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py:34]: Space after "return" (as in line 76).

File: lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+
+            print(response.read().decode())
+
+            responses.append(response)
+
+        print(responses)
+    except KeyError as e:
+        logging.exception(e)
+        # Handle failed env variable
+        print(f'Failed to find environmental variable: {e}')
```
Comment (kalbfled) [inline lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py:56]: You can include this in the call to logging.exception above.

**Review by kalbfled — CHANGES_REQUESTED** [review]
This code has problems that need to be fixed.  It should not have been merged in its current state.

**Pull request merged**